platemate/management/models/turk.py

```diff
@@ -91,17 +91,24 @@ def work_time(self):
     # Returns True if all responses are valid
     @property
     def valid(self):
-        for r in self.responses.all():
+        responses = self.responses.all()
+        if not responses:
+            return False
+        for r in responses:
             if not r.valid:
                 return r.valid
         return True
 
     @property
     def feedback(self):
-        for r in self.responses.all():
-            if r.valid is False:
-                return r.feedback
-        return self.responses.all()[0].feedback
+        responses = self.responses.all()
+        if not responses:
+            return "No response found."
```

Comment:
This will be shown to the user as a reject reason so perhaps "You didn't provide any response."